terraform/modules/data-share-service/cloudwatch.tf

```diff
@@ -41,23 +41,43 @@ module "metrics_dashboard" {
       stat   = "Sum",
       metrics = [
         {
-          name       = "API_CALLS.PublishEvent.count",
-          dimensions = { success = true },
+          name = "count.eventsController.publishEvent.count",
+          dimensions = {
+            result    = "success",
+            exception = "none",
+            method    = "publishEvent",
+            class     = "uk.gov.gdx.datashare.controllers.EventsController"
+          },
           attributes = { label = "PublishEvent" }
         },
         {
-          name       = "API_CALLS.GetEvent.count",
-          dimensions = { success = true },
+          name = "count.eventsController.getEvent.count",
```

Comment:
Is this different from `http.server.requests.count` where `exception = 'none' and method = 'GET' and uri = '/events' and outcome = 'SUCCESS'`?